Standards/scs-XXXX-v1-default-rules-for-security-groups.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Default Rules for Security Groups
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+Security Groups in IaaS (OpenStack) are part of the network security mechanisms provided for the users.
+They resemble sets of virtual firewall rules allowing specific network traffic at a port of a VM that connects it to a network.
+They are project-bound, which means that all Security Groups that are newly created are only available to the project in that they were created.
+This is also the case for the default Security Group that is created for each project as soon as the project itself is created.
+
+## Terminology
+
+Security Group (abbr. SG)
+  Set of ip table rules, used for tenant network security.
+
+Security Group Rule (abbr. SG Rule)
+  A single ip table rule, that is part of a SG.
+
+Administrator (abbr. Admin)
+  Operator = User of an OpenStack cloud with the admin role.
```

Comment:
Do we need to distinguish this from the domain admin?

Reply:
I am a bit out of the loop.

So what I'm basically asking is this: If there is conceivably more than one role which could be addressed by "admin", it would be good to clarify which admin is referred to (and if for instance the "domain-scoped manager" (which I personally could see as an admin of some kind) does not have the necessary permissions to do this, then it's worth mentioning that I think).

Reply:
Works for me.